test/repl.jl

```diff
@@ -550,11 +550,11 @@ end
 
         newname = "NewName"
         set_name(projfile_path, newname)
-        @test Pkg.REPLMode.promptf() == "($env_name) pkg> "
+        @test Pkg.REPLMode.promptf() == "($newname) pkg> "
```

Comment:
@00vareladavid can you explain these tests. Why should it not be `newname` here? We changed the name in the current project file? What am I missing.

Reply:
Yes, I wrote these tests just after `activate` was introduced. Right before, we were still using `pwd` to determine the active project. I did comment on it (because I thought it was weird) [here](https://github.com/JuliaLang/Pkg.jl/pull/466#issuecomment-402837253). In hindsight, I should have made a bigger fuss about it.